command.lua

```diff
@@ -36,7 +36,7 @@ local function buildImageProcessingCommand(config, file, flags)
     end
 
     -- Create Canvas
-    command = command .. ' -size $(identify -ping -format "%wx%h" ' .. originalFilePath .. ')'
+    command = command .. ' -size $(' .. config.identify .. ' -ping -format "%wx%h" ' .. originalFilePath .. ')'
```

Comment:
i just looked at the IM doc. looks like we can use config.magick instead of config.identify.

Reply:
I was wrong :smile: